test/plugin.js

```diff
@@ -144,6 +144,31 @@ describe("nemo-drivex @plugin@", function () {
     });
   });
   /**
+   * waitForElementVisible Wait for timeout milliseconds for the WebElement to be visible
+   * @param locator {LocatorJSON}
+   * @param timeout {Number}
+   * @param msg {String} optional message for any error messages
+   * @returns {Promise} resolves to true or false
+   */
+  describe("@waitForElementVisible@ method", function() {
```

Comment:
I think it'd be good to make the positive test case verify that there is latency while waiting for a DOM element to be visible. You can use this test as a guide:
https://github.com/paypal/nemo-view/blob/master/test/methods.js#L80-L98